Doc/library/sqlite3.rst

```diff
@@ -624,9 +617,10 @@ Cursor Objects
 
    .. method:: executemany(sql, seq_of_parameters)
 
-      Executes an SQL command against all parameter sequences or mappings found in
-      the sequence *seq_of_parameters*.  The :mod:`sqlite3` module also allows
-      using an :term:`iterator` yielding parameters instead of a sequence.
+      Executes a :ref:`parametrized <sqlite3-placeholders>` SQL command
```

Comment:
spelling
```suggestion
      Executes a :ref:`parameterized <sqlite3-placeholders>` SQL command
```

Reply:
Actually, both forms are allowed in US english, but _parameterized_ is already used in this doc, so I'll change it for consistency. Thanks :)

Reply:
I learnt something new again today. Thanks!